src/core/server/http/http_service.ts

```diff
@@ -169,81 +215,11 @@ export class HttpService
   }
 
   public async stop() {
-    if (this.configSubscription === undefined) {
```

Comment:
__note:__ that's actually not needed and also was preventing preboot server started in `preboot` from stopping if the root needed to shut down before `setup`.